core/src/cubos/core/gl/grid.cpp

```diff
@@ -81,22 +75,26 @@ const glm::uvec3& Grid::getSize() const
 
 void Grid::clear()
 {
-    for (size_t i = 0; i < this->indices.size(); i++)
-        this->indices[i] = 0;
+    for (unsigned short& indice : this->indices)
```

Comment:
You can use `fill` instead, this replaces existing elements as well:
```suggestion
    this->indices.fill(0);
```